core/types/transaction.go

```diff
@@ -53,6 +53,26 @@ const (
 	SetCodeTxType    = 0x04
 )
 
+type TxTypeBitVec [(SetCodeTxType + 1 + 7) / 8]byte
```

Comment:
The maximum tx type value is `127`, so we will need at most 127 bits. An underlying type of `[2]uint64` could be used here and it'll be good for all time.

There is also no need to make this type public.